app/js/handlebarsScoreTable.ts

```diff
@@ -411,8 +411,20 @@ function addEventListenersForMoreInfoButtons() {
             document.querySelector('tr[data-id="' + userId + '"]').classList.toggle('hide')
             if(this.textContent == 'More info') {
                 this.textContent = 'Less info';
+                e.target.parentElement.parentElement.style.background = "#fff";
+                let percentageElems = document.querySelectorAll('tr[data-percentage]')
+                percentageElems.forEach(function (percentageElem) {
+                    if (Number(percentageElem.getAttribute("data-percentage")) >= 97) {
+                        percentageElem.classList.add('top-grade')
+                    } else if (Number(percentageElem.getAttribute("data-percentage")) >= 70) {
+                        percentageElem.classList.add('passing-grade')
+                    } else {
+                        percentageElem.classList.add('dubious-grade')
```

Comment:
this code actually applies the colours to all the percentage elems on the page. not just the ones related to the current info button.

but it runs within a foreach loop (line 408), which means that this code is repeated unnecessarily multiple times (it runs once per every more info button)

you can move this code outside the foreach loop

Reply:
done!